vars/getDAOSPackages.groovy

```diff
@@ -26,7 +26,7 @@ String call(String distro, String next_version, String add_daos_pkgs) {
 
     String pkgs
     if (env.TEST_RPMS == 'true') {
-        pkgs = "daos{,-{client,tests,server,serialize}" + add_daos_pkgs + "}"
+        pkgs = "daos{,-{client,tests-internal,server,serialize}" + add_daos_pkgs + "}"
```

Comment:
Is this creating an issue that once this is landed, CI will be broken until all DAOS branches and PRs are creating these packages?

Reply:
The daos branches will need to be updated to produce this new package first.

Reply:
Yes.  There is a discussion in the PR starting at https://daosio.atlassian.net/browse/DAOS-10535?focusedCommentId=97808